server.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const path = require('path');
+const PORT = process.env.PORT || 3000;
+const atelierAPI = require('./helpers/atelierAPI.js');
+
```

Comment:
lets make a date to work on this file together so we can all be on the same page before the weekend

Reply:
Ok!